src/webui/www/public/scripts/dynamicTable.js

```diff
@@ -807,6 +807,7 @@ var TorrentsTable = new Class({
                     case "unknown":
                     case "error":
                     case "missingFiles":
+                    default:
```

Comment:
@Chocobo1, @Piccirello
Damn!
Really this `default` breaks the logic! Initially it transformed some extra states to the simple, e.g. `forcedDL` => `downloading`, `metaDL` => `downloading`, `forcedUP` => `uploading` etc. so simple states was expectedly omitted. Now we have "errored" icon for these states.
This `default` should be removed back and the appropriate comment should be added here!

Reply:
I can't reproduce.
Can you post steps?

Reply:
The issue occurs when the state is already one of the normalized values: downloading, stalledDL, uploading, etc. It won't match any of the cases, thus triggering the default case.

I've addressed this in #8304.